Update document (#1388)
infiniflow/infinity: docs/getstarted/deploy_infinity_server.mdx

### Deploy Infinity using Docker 

#### Prerequisites

- CPU: x86_64 with AVX2

> - Linux with glibc &ge; 2.17 or Windows 10+ with WSL/WSL2 or MacOS

Reviewer: ```suggestion
- Linux with glibc &ge; 2.17, Windows 10+ with WSL/WSL2, or MacOS
```

## Connect to Infinity server

> Before run following Python scripts, please make sure you have already installed infinity-sdk.

Reviewer: ```suggestion
Before running the following Python scripts, please ensure that you have already installed infinity-sdk.
```
